### DISCUSSION TITLE
What is the detrend_df_f function

### DISCUSSION THREAD

**Discussion by qweasdf1354** [nel-lab/mesmerize-core#293] [Q&A]
Hi, kushal kolar! nice to see you!

I want to ask a basic question

how to understand the detrend_df_f  function i see the original article(elife) and is difficult for me to understand the function. in my opinion the dfof can be simply understand by the F(t)-F(0)/F(0)，and i want to know the F(0) in this detrend_df_f  function mean. is that the dynamic range of baseline for the specific F(t)? or static or markered? i mean how to define the F(0), is the first 500 frame value or the 500 frame around the F(t)? Thanks a lot!

**Comment by kushalkolar** [comment]
This is a caiman question rather than a mesmerize-core one, mesmerize only provides a framework for using caiman so I would ask over there: https://github.com/flatironinstitute/CaImAn/discussions

**Comment by qweasdf1354** [comment]
Thanks Kushal! I ask in the CalmAn discussions!
